### PR TITLE
[FrameworkBundle][PropertyInfo] Move PropertyInfoPass to the PropertyInfo component

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | yes
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

**Review by ogizanagi — APPROVED** [review]

File: src/Symfony/Component/PropertyInfo/DependencyInjection/PropertyInfoPass.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+{
+    use PriorityTaggedServiceTrait;
+
+    private $propertyInfoService;
+    private $listExtractorTag;
+    private $typeExtractorTag;
+    private $descriptionExtractorTag;
+    private $accessExtractorTag;
+
+    public function __construct($propertyInfoService = 'property_info', $listExtractorTag = 'property_info.list_extractor', $typeExtractorTag = 'property_info.type_extractor', $descriptionExtractorTag = 'property_info.description_extractor', $accessExtractorTag = 'property_info.access_extractor')
```
Comment (GuilhemN) [inline src/Symfony/Component/PropertyInfo/DependencyInjection/PropertyInfoPass.php]: Is this kind of extension point really needed?

Comment (chalasr) [inline src/Symfony/Component/PropertyInfo/DependencyInjection/PropertyInfoPass.php]: Yes I think, we did it for each other (see https://github.com/symfony/symfony/pull/20250#issuecomment-254891730)

Comment (ogizanagi) [inline src/Symfony/Component/PropertyInfo/DependencyInjection/PropertyInfoPass.php]: Why not? It allows configuring multiple property info services in an application easily.

Comment (GuilhemN) [inline src/Symfony/Component/PropertyInfo/DependencyInjection/PropertyInfoPass.php]: It adds stuff to maintain and I'm not sure it's really useful, but if you think it is why not.

**Comment by fabpot** [comment]
Thank you @chalasr.

**Pull request merged**